test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
Thanks @Praburaj 
I found MusicStore don't use DI for AntiForgery. 
https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/Controllers/ShoppingCartController.cs#L74-L75

I'm wondering if I could change this to use the property injection like the other services, cache and dbcontext??